### PR TITLE
fix(regions): ignore enterprise kafkas when checking for regions capacity

### PR DESCRIPTION
**Pull request by machi1990**
## Description

Enterprise cluster don't eat up regions capacity and in that regard they should be omitted when we check if region's limits have been reached.

## Verification Steps

Unit & integration test should pass

## Checklist (Definition of Done)

- [ ] All acceptance criteria specified in JIRA have been completed
- [ ] Unit and integration tests added that prove the fix is effective or the feature works (tested against emulated and non-emulated OCM environment)
- [ ] Documentation added for the feature
- [ ] CI and all relevant tests are passing
- [ ] Code Review completed
- [ ] Verified independently by reviewer
- [ ] All PR comments are resolved either by addressing them or creating follow up tasks
- [ ] Required metrics/dashboards/alerts have been added (or PR created).
- [ ] Required Standard Operating Procedure (SOP) is added.
- [ ] JIRA has been created for changes required on the client side

### CODE REVIEW

**Comment by machi1990** [comment]
@henryZrncik this should fix the issue you observed in Stage

File: internal/kafka/test/integration/cloudprovider_test.go [hunk 1]
```diff
@@ -465,8 +478,8 @@ func TestListCloudProviderRegions(t *testing.T) {
 		}
 	}
 
-	// Create a kafka in us-east-1 to use up capacity
-	err = test.TestServices.DBFactory.New().Create(&dbapi.KafkaRequest{
+	// Create a non enteprise kafka in us-east-1 to use up capacity
```
Comment (pawelpaszki) [inline internal/kafka/test/integration/cloudprovider_test.go:481]:
```suggestion
	// Create a non enterprise kafka in us-east-1 to use up capacity
```

File: internal/kafka/test/integration/cloudprovider_test.go [hunk 2]
```diff
@@ -436,6 +435,20 @@ func TestListCloudProviderRegions(t *testing.T) {
 	account := h.NewRandAccount()
 	ctx := h.NewAuthenticatedContext(account, nil)
 
+	db := test.TestServices.DBFactory.New()
+
+	// Create a enteprise kafka in us-east-1 which shouldn't consume capacity
```
Comment (pawelpaszki) [inline internal/kafka/test/integration/cloudprovider_test.go:440]:
```suggestion
	// Create an enterprise kafka in us-east-1 which shouldn't consume capacity
```

**Review by pawelpaszki — APPROVED** [review]
minor typos. otherwise looks good!

**Comment by miguelsorianod** [comment]
Looks good to me

**Pull request merged**